js/editors/ace/editor-ace.js

```diff
@@ -300,20 +299,7 @@ window.AceEditor = Backbone.View.extend({
 
     // Set the cursor position on the editor
     setErrorHighlight: function(shouldHighlight) {
-        var self = this;
-
         this.editor.setHighlightActiveLine(shouldHighlight);
-
-        // Delay adding a flash until the active line is shown
-        setTimeout(function() {
```

Comment:
At one point, this code flashes a red background on the highlighted line. It no longer does that, since the ACTIVE_LINE class doesn't exist. Now the line is still highlighted gray, thanks to setHighlightActiveLine, but there's no red background.

I can bring back the red (as I know what the new class name is) if desired. However, it doesn't seem needed, given we didn't notice it was gone, plus this ace-dependent code isn't the most stable.